File: SurakshaNet/Mobile/suraksha_net_app/lib/shared/models/incident.dart
class Incident {
  Incident({required this.id, required this.category, required this.title, required this.status, required this.severity, this.description, this.isAnonymous = true});

  final int id;
  final String category;
  final String title;
  final String status;
  final String severity;
  final String? description;
  final bool isAnonymous;
}

File: Server/SurakshaNet.Api/Models/Incident.cs
namespace SurakshaNet.Api.Models;

public sealed class Incident : BaseEntity
{
    public string Title { get; set; } = string.Empty;

    public string Description { get; set; } = string.Empty;

    public string Category { get; set; } = string.Empty;

    public string Status { get; set; } = "Submitted";

    public decimal ApproximateLatitude { get; set; }

    public decimal ApproximateLongitude { get; set; }

    public bool ExactLocationConsent { get; set; }

    public Guid? ReporterUserId { get; set; }

    public User? ReporterUser { get; set; }
}
